### ISSUE TITLE
Joomla code style

### ISSUE THREAD

**Issue by rinka88** [DEVSENSE/phptools-docs#395]
Hello Will there be code style support for joomla 4 and 5 versions?

**Comment by Miloslav** [comment]
Absolutely! It appears the Joomla project adheres to PSR standards, so it shouldn't be too difficult.

Do you know any particular deviations from the PSR we should be aware of?

Thanks
